src/xdocs/config_imports.xml

```diff
@@ -1318,8 +1319,9 @@ import java.util.stream.IntStream;
           </tr>
           <tr>
             <td>sortStaticImportsAlphabetically</td>
-            <td>whether static imports grouped by <b>top</b> or <b>bottom</b> option
-            are sorted alphabetically or not</td>
+            <td>whether <b>static imports</b> grouped by <b>top</b> or <b>bottom</b> option
+                are sorted alphabetically or not.
+                Attention:it is applied for all static imports as one group.</td>
```

Comment:
=> `Attention: It is applied to all static imports as one group.`

Reply:
Done.